big_tests/tests/mod_blocking_SUITE.erl

```diff
@@ -647,7 +649,6 @@ message_is_delivered(From, To, MessageText) ->
 message_is_not_delivered(From, [To|_] = Tos, MessageText) ->
     BareTo = escalus_utils:jid_to_lower(escalus_client:short_jid(To)),
     escalus:send(From, escalus_stanza:chat_to(BareTo, MessageText)),
-    timer:sleep(300),
```

Comment:
I am afraid that this test would pass even if the message is delivered...

Reply:
I've pushed a new commit introducing a sleep in the only test where this would have an effect. In all other tests, more messages are being sent and received after they call `message_is_not_delivered/3`, so if the message that wasn't supposed to arrive here arrives there, the test will fail with a different assertion, but it would fail. For the most part, it makes CI faster to have as little sleeps as possible.